src/utils/CatchErorr.js

```diff
@@ -0,0 +1,9 @@
+export const catchError = (action) => {
```

Comment:
There is a typo in the file name. It should be `CatchError.js` instead of `CatchErorr.js` to maintain consistency with the import statements.